docs/workflows/genomic_characterization/theiaviral.md

```diff
@@ -85,7 +85,7 @@
 /// html | div[style='float: left; width: 50%; padding-right: 10px;']
 
 ??? dna "`host` optional input parameter"
-    The `host` input triggers the Host Decontaminate workflow, which removes reads that map to a reference host genome. This input needs to be an [NCBI Taxonomy-compatible](https://www.ncbi.nlm.nih.gov/taxonomy) taxon or an NCBI assembly accession. If using a taxon, the first retrieved genome corresponding to that taxon is retrieved. If using an accession, it must be coupled with the Host Decontaminate task `is_accession` (ONT) or Read QC Trim PE `host_is_accession` (Illumina) boolean populated as "true".
+    The `host` input triggers the Host Decontaminate workflow, which removes reads that map to a reference host genome. This input needs to be an [NCBI Taxonomy-compatible](https://www.ncbi.nlm.nih.gov/taxonomy) taxon, host genome Google storage URI, or an NCBI assembly accession. If using a taxon, the first retrieved genome corresponding to that taxon is retrieved. If using a genome URI or accession, it must be coupled with the Host Decontaminate task  `is_genome`/`is_accession` (ONT) or Read QC Trim PE `host_is_genome`/`host_is_accession` (Illumina) boolean populated as "true".
```

Comment:
couldn't host just be any random assembly fasta? could even be an existing column in your table